ScreenConfig.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+import "brewblox.proto";
+import "nanopb.proto";
+
+package blox.ScreenConfig;
+
+enum Type {
+  Row = 0;
+  Column = 1;
+  Content = 2;
+}
+
+message LayoutNode {
+  uint32 parent = 1 [(nanopb).int_size = IS_8];
+  uint32 nodeId = 2 [(nanopb).int_size = IS_8];
+  Type type = 3;
+  uint32 weight = 4 [(nanopb).int_size = IS_16];
+}
+
+message Color {
```

Comment:
Overly generic name.

Reply:
I don't agree it is overly generic cause it lives in the namespace of `screenconfig`. It represents a color hence it's name. Would you name it differently?

Reply:
It only includes RGB, so chances are it can be re-used if more color configuration appears within this context. Leave as-is then.